api/test/robot_tests/web_on_chain/web_on_chain.robot

```diff
@@ -46,14 +46,26 @@ POST /websiteUploader/upload
 
     Should Be Equal As Strings    ${response.json()['address']}    ${WEBSITE_ADDRESS}
 
-GET /all/domains 
-    ${response}=    GET
-    ...    ${API_URL}/all/domains
-    ...    expected_status=${STATUS_OK}
-    Should Contain    ${response.text}    flappy
-
 GET /my/domains/{nickname}
     ${response}=    GET
     ...    ${API_URL}/my/domains/${WALLET_NICKNAME}
     ...    expected_status=${STATUS_OK}
     Should Contain    ${response.text}    ${website_url}
+    
+POST /cmd/executeFunction To remove the dns entry
+    ${argument}=    keywords.String To Arg    ${website_url}
+    ${data}=    Create Dictionary
+    ...    nickname=${WALLET_NICKNAME}
+    ...    name=deleteEntryFromDNS
+    ...    at=AS12tsy5TtDyGsWggDvqDLagxZqkuTiTPqWhqmrx7KSQNJwQSbnBA
+    ...    args=${argument}
+    ${response}=    POST
+    ...    ${API_URL}/cmd/executeFunction
+    ...    json=${data}
+    ...    expected_status=${STATUS_OK}
+    
+GET /all/domains 
+    ${response}=    GET
+    ...    ${API_URL}/all/domains
+    ...    expected_status=${STATUS_OK}
+    Should Contain    ${response.text}    flappy
```

Comment:
why did you move this test at the end?

Reply:
i moved this one GET /my/domains/{nickname} because it check if the ci successfully added the new dns entry , so you want to remove a dns entry  after this test GET /my/domains/{nickname} but the order of GET /all/domains 
 is irrelevant